mlrun/feature_store/api.py

```diff
@@ -62,9 +63,11 @@
     run_spark_graph,
 )
 from .retrieval import get_merger, init_feature_vector_graph, run_merge_job
+from .steps import DropFeatures
 
 _v3iofs = None
 spark_transform_handler = "transform"
+ENTITIES_DROPPED_MESSAGE = "Dropping features is illegal on entities keys"
```

Comment:
"DropFeatures can only drop features, not entities"

Reply:
ty!